Tests/AWSLambdaEventsTests/S3Tests.swift

```diff
@@ -85,4 +126,32 @@ class S3Tests: XCTestCase {
         XCTAssertEqual(record.s3.object.eTag, "91a7f2c3ae81bcc6afef83979b463f0e")
         XCTAssertEqual(record.s3.object.sequencer, "005E1C37948E783A6E")
     }
+    
+    func testObjectRemovedEvent() {
+        let data = S3Tests.eventBodyObjectRemoved.data(using: .utf8)!
+        var event: S3.Event?
+        XCTAssertNoThrow(event = try JSONDecoder().decode(S3.Event.self, from: data))
+
+        guard let record = event?.records.first else {
+            XCTFail("Expected to have one record")
+            return
+        }
+
+        XCTAssertEqual(record.eventVersion, "2.1")
+        XCTAssertEqual(record.eventSource, "aws:s3")
+        XCTAssertEqual(record.awsRegion, .eu_central_1)
+        XCTAssertEqual(record.eventName, "ObjectRemoved:DeleteMarkerCreated")
+        XCTAssertEqual(record.eventTime, Date(timeIntervalSince1970: 1_578_907_540.621))
+        XCTAssertEqual(record.userIdentity, S3.UserIdentity(principalId: "AWS:AAAAAAAJ2MQ4YFQZ7AULJ"))
+        XCTAssertEqual(record.requestParameters, S3.RequestParameters(sourceIPAddress: "123.123.123.123"))
+        XCTAssertEqual(record.responseElements.count, 2)
+        XCTAssertEqual(record.s3.schemaVersion, "1.0")
+        XCTAssertEqual(record.s3.configurationId, "98b55bc4-3c0c-4007-b727-c6b77a259dde")
+        XCTAssertEqual(record.s3.bucket.name, "eventsources")
+        XCTAssertEqual(record.s3.bucket.ownerIdentity, S3.UserIdentity(principalId: "AAAAAAAAAAAAAA"))
+        XCTAssertEqual(record.s3.bucket.arn, "arn:aws:s3:::eventsources")
+        XCTAssertEqual(record.s3.object.key, "Hi.md")
```

Comment:
Maybe we want to explicitly check that `object.size` is nil?
```suggestion
        XCTAssertEqual(record.s3.object.key, "Hi.md")
        XCTAssertNil(record.s3.object.size)
```